ledger/ledger-api-bench-tool/src/test/suite/scala/com/daml/ledger/api/benchtool/config/WorkflowConfigParserSpec.scala

```diff
@@ -32,7 +32,8 @@ class WorkflowConfigParserSpec extends AnyWordSpec with Matchers {
           |         - Foo1
           |         - Foo3
           |    objectives:
-          |      min_item_rate: 123""".stripMargin
```

Comment:
All tests in this file have `min_item_rate` and `max_item_rate` defined. Is the idea that those members are mandatory? If yes, remove the wrapping `Option`. If no, add tests with those members missing.

Related, do you want to support a mode where there are no objectives? I have used the benchtool before to measure the ACS performance where I did not have any objectives in mind. If found it a bit awkward that I had to add the `objectives` struct with some random values in it.

Reply:
> All tests in this file have min_item_rate and max_item_rate defined. Is the idea that those members are mandatory? If yes, remove the wrapping Option. If no, add tests with those members missing.

All objectives are meant to be optional. I'll add appropriate tests for cases without objectives - thank you for spotting this 👍 

> Related, do you want to support a mode where there are no objectives? I have used the benchtool before to measure the ACS performance where I did not have any objectives in mind. If found it a bit awkward that I had to add the objectives struct with some random values in it.

I'll make them truly optional in configuration as well.

Reply:
Optional objectives: 2523e32f9509060ab1e7d791dd53c2da19befffb
Unit tests: c24d6410b72162233776fd3a5ee4ae63d099a4a5